### DISCUSSION TITLE
QBtnToggle scoped slot

### DISCUSSION THREAD

**Discussion by FragsterAt** [quasarframework/quasar#15618] [Ideas / Proposals] [answered]
It would be very nice if QBtnToggle would have a slot with a scope, for example #item="scope", which would have a element from the options array and an index, inside it would be possible to place QBtn. The current reality forces to produce slots for monotonous buttons. For example, an example from the [documentation ](https://github.com/quasarframework/quasar/blob/dev/docs/public/examples/QBtnToggle/CustomContent.vue) would be solved by specifying the name of the icon in the options, and not by creating three almost identical slots.

**Comment by jruuttun** [comment, marked as answer]
I just learned something about the q-button-toggle customization from [this video](https://www.youtube.com/watch?v=R27lXvxaRoY), maybe it could help you too? Note that you could create the slot templates by `v-for`-iterating your options, like in the [codepen](https://codepen.io/smolinari/pen/yLyMErW?editors=1010) referenced in #5925. Btw the documentation link in the description is broken.

**Reply by FragsterAt** [reply]
Nice workaround! Thanks!
